src/harness/testcases/WINNF_FT_S_PAT_testcase.py

```diff
@@ -104,25 +103,43 @@ def test_WINNF_FT_S_PAT_1(self, config_filename):
     
     config = loadConfig(config_filename)
     num_tests = len(config)
-    test_pass_threshold = 1.00
-    passed_tests = 0.0
-    for configItem in  config:
-       request= config[configItem]
-       refResponse = computePropagationAntennaModel(request)
-       sasResponse = self._sas_admin.QueryPropagationAndAntennaModel(request)
-
-       # Check response
-       this_test = False
-       if 'pathlossDb' in refResponse and 'txAntennaGainDbi' in refResponse:
-           this_test = (sasResponse['pathlossDb'] < refResponse['pathlossDb'] + 1) and (sasResponse['txAntennaGainDbi'] < (refResponse['txAntennaGainDbi'] + .2))
-               
-           if 'rxAntennaGainDbi' in refResponse:
-               this_test = this_test and (sasResponse['rxAntennaGainDbi'] < (refResponse['rxAntennaGainDbi'] + .2))
-       else:
-           self.assertEqual(sasResponse, refResponse)
-           this_test = 1.0
+    test_pass_threshold = 0.999 #percentage of passed test expected
+    num_passed_tests = 0.0
+    num_tests = len(config)
+    max_fail_num = int ((1-test_pass_threshold) * num_tests)
+    num_failed_tests = 0.0
+    num_invalid_tests = 0.0
+    for request in  config:
+        try:
+           refResponse = computePropagationAntennaModel(request)
+           sasResponse = self._sas_admin.QueryPropagationAndAntennaModel(request)
 
-       passed_tests += this_test*1.0
+                  # Check response
+           this_test = False
```

Comment:
I think you need to set this to `True`, otherwise the SAS will always fail even if it gives the correct response.